### PR TITLE
Update submission components for v1.2

### PR DESCRIPTION
**Pull request by matthew-white**

### CODE REVIEW

File: src/components/submission/feed-entry.vue [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  }
+
+  .icon-cloud-upload, .icon-clock-o { color: #bbb; }
+  .review-state {
+    color: #999;
+    &.hasIssues { color: $color-warning; }
+    &.edited { color: #666; }
+    &.approved { color: $color-success; }
+    &.rejected { color: $color-danger; }
+  }
```
Comment (matthew-white) [inline src/components/submission/feed-entry.vue:119]: Change any icon colors?

File: src/components/submission/feed-entry.vue [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+  time {
+    float: right;
+    font-size: 16px;
+    color: #666;
+  }
+
+  .title {
+    @include text-overflow-ellipsis;
+    font-size: 18px;
```
Comment (matthew-white) [inline src/components/submission/feed-entry.vue:99]: Decrease the font size?

File: src/components/submission/feed-entry.vue [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+https://github.com/getodk/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+<template>
+  <div class="submission-feed-entry">
```
Comment (matthew-white) [inline src/components/submission/feed-entry.vue:13]: Should there be an effect if the entry corresponds to an edit without a comment?

File: src/components/submission/feed-entry.vue [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    }
+  }
+};
+</script>
+
+<style lang="scss">
+@import '../../assets/scss/mixins';
+
+.submission-feed-entry {
+  box-shadow: 0 0 6px rgba(0, 0, 0, 0.09);
```
Comment (matthew-white) [inline src/components/submission/feed-entry.vue:85]: Is this the right box shadow?

**Review by matthew-white — COMMENTED** [review]
Notes before interactive review

File: src/components/submission/comment.vue [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+    }
+  }
+};
+</script>
+
+<style lang="scss">
+@import '../../assets/scss/mixins';
+
+#submission-comment {
+  @include clearfix;
```
Comment (matthew-white) [inline src/components/submission/comment.vue:104]: Move to the actions div?

**Review by issa-tseng — APPROVED** [review]

**Pull request merged**